pyserini/search/lucene/irst/_searcher.py

```diff
@@ -87,6 +89,55 @@ def from_prebuilt_index(cls, prebuilt_index_name: str):
         print(f'Initializing {prebuilt_index_name}...')
         return cls(index_dir)
 
+    def download_and_unpack_translation_model(self):
```

Comment:
there seems to be a lot of code duplication here with `pyserini/util.py` - is it really necessary? why can't you just reuse the existing methods?

Reply:
All CR fixed.